code_samples/front/shop/shipping/config/packages/services.yaml

```diff
@@ -0,0 +1,38 @@
+services:
```

Comment:
Should be in `config/packages/services.yaml` but `config/services.yaml`, right? (It's just internal organisation, Markdown text got it right)

Reply:
Moved in #2097